userspace/libsinsp/socket_handler.h

```diff
@@ -1561,14 +1553,21 @@ class socket_data_handler
 			{
 				*(parser_data->m_msg_completed) = true;
 			}
-			else { throw sinsp_exception("Socket handler: parser msg complete null."); }
+			else { throw sinsp_exception("Socket handler (http_msg_completed_callback): parser data m_msg_completed is null."); }
+			if(parser_data->m_http_response)
+			{
+				*(parser_data->m_http_response) = parser->status_code;
+			}
+			else { throw sinsp_exception("Socket handler (http_msg_completed_callback): parser data m_http_response is null."); }
 		}
-		else { throw sinsp_exception("Socket handler: parser or data null."); }
+		else { throw sinsp_exception("Socket handler (http_msg_completed_callback): parser or data null."); }
 		return 0;
 	}
 
 	void init_http_parser()
```

Comment:
Github mangled which line this was for. My comment was about calling init_http_parser() multiple times.